python/cudf/cudf/core/reshape.py

```diff
@@ -201,44 +203,46 @@ def concat(objs, axis=0, join="outer", ignore_index=False, sort=None):
     1      b       2  monkey  george
     """
 
+    # TODO: Do we really need to have different error messages for an empty
+    # list and a list of None?
```

Comment:
The answer to this is probably no but  we usually to just follow whatever pandas does. 
```
>>> pd.concat([])
ValueError: No objects to concatenate
>>> pd.concat([None, None, None])
ValueError: All objects passed were None
```